lib/SVF-FE/PAGBuilder.cpp

```diff
@@ -116,6 +116,22 @@ PAG* PAGBuilder::build(SVFModule* svfModule)
         }
     }
 
+    // fill valuToEdgeMap
+    unsigned edge_count = 0;
+    unsigned node_count = 0;
+    for (auto& node : *pag) {
+        node_count++;
+        for (auto edge_it = node.second->OutEdgeBegin(); edge_it != node.second->OutEdgeEnd(); ++edge_it) {
+            auto edge = *edge_it;
+            pag->mapValueToEdge(edge->getValue(), edge);
+            edge_count++;
+        }
+    }
+    errs() << "NodeCount" << node_count << " NodeNum: " << pag->getTotalNodeNum() << " " << pag->getPAGNodeNum() << "\n";
```

Comment:
delete these output messages.

Reply:
I have inserted these messages only because the assertion a few lines later does not work. Can you elaborate on that?
Why is `getTotalEdgeNum()` different to `edge_count`?

Reply:
It is strange. Could you have a look at where these two counters `getTotalEdgeNum()` and `edge_count` are increamented?